Improvements

paodelonga opened the pull request:
- Update app/bad_words.json
> Foram adicionados outros palavrões e dezenas de suas variações

- Update in app/commands/commands.js
> Atualização feita nas mensagens enviadas pelo bot
quando alguém digita algum "palavrão".
Além da mensagem padrão agora é possível enviar
outras mensagens de forma randomica;
as respostas podem alteradas em app/warnin_messages.json

- Added app/warnin_messages.json
> Arquivo que contem as mensagens de resposta do bot

paodelonga committed:
Improvements

- Update app/bad_words.json
> Foram adicionados outros palavrões e dezenas de suas variações

- Update in app/commands/commands.js
> Atualização feita nas mensagens enviadas pelo bot
quando alguém digita algum "palavrão".
Além da mensagem padrão agora é possível enviar
outras mensagens de forma randomica;
as respostas podem alteradas em app/warnin_messages.json

- Added app/warnin_messages.json
> Arquivo que contem as mensagens de resposta do bot

fabegalo commented:
Opa @paodelonga , temos 3 problemas no commit, 
1) - Não está puxando o conteudo do warning_messages.json
2) - Ele está puxando fixamente como Texto
3) - No arquivo bad_words.json verificar porque ele está filtrando todas as palavras, mesmo as que não são palavrões

fabegalo reviewed (commented):
![image](https://user-images.githubusercontent.com/53371411/152699738-c49d3ed7-0d3d-4b8f-b07a-fa6b5408d862.png)
Correção 1)

fabegalo commented:
![image](https://user-images.githubusercontent.com/53371411/152699876-8e3eaf4b-c3fc-43d2-a915-adb68598bb64.png)

Correção 3)

paodelonga commented:
Pode deixar, já estou aplicando as correções.

paodelonga commented:
Algumas correções já foram aplicadas:
* O Conteúdo do `warning_messages.json` está sendo puxado (Havia também um pequeno erro no nome do arquivo).
porém ainda continua vindo como apenas texto, logo mais irei aplicar a devida correção.
* O Arquivo `bad_words.json` foi corrigido!
infelizmente o problema estava não só no arquivo, mas, sim no meu código que gerou o arquivo, e com isso pude lançar a correção também ao meu código.

_Obrigado pelo feedback, Nice Coding!_ :grinning:

paodelonga committed:
Pequenas correções.

paodelonga commented:
Algumas correções já foram aplicadas:
* O Conteúdo do `warning_messages.json` está sendo puxado (Havia também um pequeno erro no nome do arquivo).
porém ainda continua vindo como apenas texto, logo mais irei aplicar a devida correção.
* O Arquivo `bad_words.json` foi corrigido!
infelizmente o problema estava não só no arquivo, mas, sim no meu código que gerou o arquivo, e com isso pude lançar a correção também ao meu código.

@fabegalo _Obrigado pelo feedback, Nice Coding!_ :grinning:

paodelonga committed:
Pequenas melhorias.

- Alterações em "badWords.json".
Foram adicionadas algumas novas palavras neste arquivo e outras foram removidas.
> _Ainda faltam correções nescessárias, algumas se encontram em falta, duplicatas ou estão escrita de forma digamos que "incorreta"._

paodelonga committed:
Melhorias, correções e/ou pequenos ajustes no código.

- Renomeado `newCommandos.json` para `slashCommands.json`
Pequenos detalhes e uma mudança na mensagem que era retornada ao usuário quando o mesmo não havia permissão:
> _Ao invés da mensagem retornar que ele apenas não havia permissão
agora, ela também mostra qual é esta permissão._

- Toda a identação foi convertida para Tabs

paodelonga committed:
warning_messages.json renomeado para warnMessages

paodelonga committed:
Melhorias feitas em em `warnMessages.json`.

Adicionei algumas frases a mais, e nelas um parametro (`AuthorID`) (disse certo? parametro? é isso né :grin:) que servirá para que a condição que verifica os palavrões nas mensagens possa fazer
menção ao usuário.

paodelonga committed:
Update app.js

- Pequenas mudanças feitas em `app.js`

paodelonga committed:
Alterações em "commands.js" e "config.json

- Corrigida a condição que verifica se há um palavrão nas mensagens
dos usuários e ao identificar a mensagem é deletada e o usuário é advertido.
> A mensagem/adventência é dada conforme nas mensagens que estão em `warnMessages.json`

- Agora os antigos comandos possuem aliases!
Foi feita uma nova forma forma de adicionar os comandos, e agora é possível
adicionar apelidos/nomes alternativos para o mesmo comando de forma simples.

**Para criar um novo comando:**
```js
if (["nome_do_comando"]).includes(command)) {
	// Faz alguma coisa...
	}
```

**Para adicionar um "apelido":**
```js
if (["nome_do_comando", "apelido"]).includes(command)) {
	// Faz alguma coisa...
	}
```

- Agora o prefixo é setado automaticamente com base nas informações de servidor que está em `configs.json`.
> Caso queira adicionar o Bot à servidores cada um deles deve ter suas informações preenchidas de forma correta
> em `configs.json`

> Se o Servidor não estiver salvo o prefixo será definido para o padrão (.).

Para adicionar vá em .configs.json em em `Servers.ID` adicione as informações.

**Template**:
```json

		"ID": {
			"GuildID": {
				"ServerName": "Nome do Servidor",
				"Prefix": "Prefixo",
				"Roles":{
					// Essa informação é opcional
					// Talvez ela seja usada futuramente.
					// Use nomes simples, sem emojis!
					"Cargo1": "RoleID",
					"Cargo2": "RoleID",
				}
			}
		}

```

### Visual
Foram feitas melhorias alterações visuais
e toda a identação do código foi convertida para "Tabs".

paodelonga committed:
Revert "Melhorias, correções e/ou pequenos ajustes no código."

This reverts commit 7a8749efdbf53be86d00e1a3719d8db86395a918.

paodelonga committed:
Revert "Alterações em "commands.js" e "config.json"

This reverts commit 54fc4a50c4f72ce4ba524e358b9c59691b8ce50d.

paodelonga committed:
Revert "Update app.js"

This reverts commit 70a6f8ca668abd397b6140f899666fef90d6fa64.

paodelonga committed:
Revert "Melhorias feitas em em `warnMessages.json`."

This reverts commit 712ae15f8ffc2233546438dda292347fddd3a82e.

paodelonga committed:
Revert "warning_messages.json renomeado para warnMessages"

This reverts commit 95bf39e60f8e0fe1bb46b3c54130206adf444f5d.

paodelonga committed:
Update .gitignore

paodelonga committed:
Rename warning_messages.jsons

paodelonga committed:
Update warnMessages.json

Adicionei algumas frases a mais, e nelas um parametro (`AuthorID`) (disse certo? parametro? é isso né :grin:) que servirá para que a condição que verifica os palavrões nas mensagens possa fazer
menção ao usuário.
